google-ads-examples/src/main/java/com/google/ads/googleads/examples/advancedoperations/AddSmartCampaign.java

```diff
@@ -194,7 +194,7 @@ private void runExample(
     // [START add_smart_campaign_14]
     // Converts the location ID to unsigned long (potentially contains IDs with the most significant
     // bit set).
-    long locationIdParsed = Long.parseUnsignedLong(locationId);
+    Long locationIdParsed = (locationId != null ? Long.parseUnsignedLong(locationId) : null);
```

Comment:
Could you write this as the positive case first? It's generally easier to mentally parse the positive case of things.

`locationIdParsed == locaitonId == null ? Long.parseUnsignedLong(locationId) : null;`

Reply:
Done.